.github/actions/beta-action/action.yml

```diff
@@ -26,15 +26,24 @@ inputs:
   chat-service-sid:
     description: 'Chat Service SID'
     required: true
-  flex-proxy-service-sid:
-    description: 'Flex Proxy Service SID'
-    required: true
   operating-info-key:
     description: 'Operating Info Key'
     required: true
   serverless-base-url:
     description: 'Serverless base url'
     required: true
+  flex-proxy-service-sid:
+    description: 'Flex Proxy Service SID'
+    required: true
+  system-secret:
```

Comment:
As with comments on https://github.com/techmatters/hrm/pull/122, this is too general.  Maybe `api-key-write-only`?  I don't like that either, but I want to be clear about what this is.